views/index.erb

```diff
@@ -1 +1,25 @@
-<h1>What am I worth</h1>
+<h1>What Am I Worth?</h1>
+
+<form class="survey" method="post">
+  <label for="email">What's your email address?
+    <input type="text" name="email" value="">
+  </label>
+  <br>
+  <label for="github">What's your GitHub account?
+    <input type="text" name="github" value="">
+  </label>
+  <br>
+  <label for="linkedin">What's your LinkedIn profile?
+    <input type="text" name="linkedin" value="">
+  </label>
+  <br>
+  Which city would you like to work in?
+    <label for="city"></label>Berlin<input type="checkbox" name="city" value="berlin">
+    <label for="city"></label>Hamburg<input type="checkbox" name="city" value="hamburg">
+    <label for="city"></label>Munich<input type="checkbox" name="city" value="munich">
+    <label for="city"></label>London<input type="checkbox" name="city" value="london">
+    <label for="city"></label>Amsterdam<input type="checkbox" name="city" value="amsterdam">
+    <label for="city"></label>Other<input type="checkbox" name="city" value="other">
```

Comment:
The `for` attribute on labels is useful because it lets the user select a form element by clicking on the label.

`for` references the `id` of a form element, so you'll have to give these checkboxes unique id attributes, and then reference those with `for`.